src/pages/Settings.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useEffect} from "react";
+import {useForm} from "antd/es/form/util";
+import {Button, Card, Form, Input, notification} from "antd";
+import {formatMessage, FormattedMessage} from "umi-plugin-react/locale";
+import {router} from "umi";
+import {PageHeaderWrapper} from "@ant-design/pro-layout";
+import {getAdminAPI, getAdminAPIKey} from "@/utils/utils";
+
+const layout = {
+  labelCol: {
+    span: 2,
+  },
+  wrapperCol: {
+    span: 8,
+  },
+};
+
+const tailLayout = {
+  wrapperCol: {
+    offset: 2,
+  },
+};
+
+const Settings: React.FC = () => {
+  const [form] = useForm();
+
+  useEffect(() => {
+    form.setFieldsValue({
+      admin_api: getAdminAPI(),
```

Comment:
snake format?

Reply:
like `get_admin_api()`?

Reply:
haha no, we use camelCase format here, so it should be `adminAPI: xxx`